Resources/Prototypes/Entities/Mobs/NPCs/human.yml

```diff
@@ -39,3 +39,11 @@
       types:
         Blunt: 200
 
+- type: entity
+  parent: MobHuman
+  id: MobCluwne
+  name: person
+  noSpawn: true
```

Comment:
i meant that this should be spawnable and without the suffix

Reply:
t shouldnt really be spawnable, like no need ot use it to spawn since it only exists so the polymorph works.

Reply:
Removed the noSpawn.